src/game/game.service.js

```diff
@@ -169,6 +169,28 @@ angular.module('ngGo.Game.Service', [
       this.setTurn((this.info.game.handicap > 1) ? StoneColor.W : StoneColor.B);
     }
 
+    /**
+     * Validate the move is not in the existed children nodes.
+     * @param x X position.
+     * @param y Y position.
+     * @param color Stone color.
+     * @returns {(null|Number)} If the move exists, return the index of the children; otherwise, return null.
```

Comment:
Could you remove the docblock style comments? I don't like using them in code. The function parameter names should make it clear what the input is, and the return value (if any) should be clear from the function name and/or general comment for the function.